src/js-form-validator.ts

```diff
@@ -356,4 +363,36 @@ export class FormValidator {
     }
     return status;
   }
+
+  private static HasValidLength(inputValue: string, minLength: number, maxLength: number): boolean {
+    if (!this.IsValidLengthValue(minLength) && !this.IsValidLengthValue(maxLength))
+      return true;
+
+    if (this.IsValidLengthValue(minLength) && inputValue.length < minLength ||
+        this.IsValidLengthValue(maxLength) && inputValue.length > maxLength)
+      return false;
+
+    return true;
+  }
+
+  private static IsValidLengthValue(value: number): boolean {
+    return !isNaN(value) && value !== 0;
+  }
+
+  private static GetInputLengthErrorMessage(inputValue: string, minLength: number, maxLength: number): string {
+    let errorMessage = '';
+
+    if (this.IsValidLengthValue(minLength) && inputValue.length < minLength)
```

Comment:
`this.IsValidLengthValue(minLength)`

This reads weird to me, are you checking to make sure the `minLength` itself is valid? If so, could this check be done when this is set instead of everytime it attempts to use it?

Reply:
@Izzmo it is checking if the value is valid. I did rename it to help clarify. I am not sure I am understanding what you mean by checking it when it is set, could you explain a little more? The intent is to verify that the number getting passed in is valid before testing it against the input length. It was a quick utility method to use so that I don't test the length if the attribute isn't even set or set to `0`.

Reply:
@someyoungideas Ah, yes I'm seeing now, static method with input, but, instead of validating the length value every time, could you do it once and throw an error back if that is invalid itself?

Reply:
@Izzmo I see what you are asking about now. This may be a fault with how I structured the method, but the reason I am checking it multiple times is because an input could have only one of the attributes set and we won't require both `minLength` and `maxLength` to be set. In the scenario that only one attribute is specified we wouldn't be able to error back because the other could be set properly. I added a single check at the top that both values are valid and return out if they are invalid.